templates/sds-bamm-aspect-model.md

```diff
@@ -0,0 +1,78 @@
+# BAMM Aspect Model Generator
+
+(C) 2022 Robert Bosch GmbH
+
+## Introduction
+
+The BAMM Aspect Model Generator converts VSC services to a valid BAMM Aspect Model according to the [BAMM Aspect Meta Model](https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model), 
+keeping as much as possible of the information from the VSC definition.  It consists of a template ([sds-bamm-aspect-model](sds-bamm-aspect-model.j2) and [sds-bamm-macros](sds-bamm-macros.j2)) 
```

Comment:
We need adapt the template file endings, here the links expect *.j2, but the files actually have *.tpl suffix. We need to make sure they use the same suffix so the links work.